terminusdb_client/woqlclient/woqlClient.py

```diff
@@ -152,31 +441,39 @@ def create_database(
         label=None,
         description=None,
         prefixes=None,
-        include_schema=True,
+        schema=True,
```

Comment:
Now that this PR is targeting `master`, should I set this back to `include_schema`? @kevinchekovfeeney

Reply:
Actually yes please do - sorry for the confusion - the include_schema flag can be mapped easily to the internal API schema flag - this is a brand new option in the latest server release so we're still working it in properly, but yes, include_schema is good.

Reply:
Thanks, changes made :slightly_smiling_face: